aries-site/src/pages/templates/inline-notifications.mdx

```diff
@@ -0,0 +1,143 @@
+import { Example } from '../../layouts';
+import { InlineNotificationExample } from '../../examples';
+import { Checkmark } from 'grommet-icons';
+
+Inline notifications are messages that appear near the content to which they
+are related. They may be task or system generated, communicating information
+such as status updates, alerts to success or failure of an action, or call
+attention to a new feature in which a user may be interested.
+
+<Example
+  code=""
+  github=""
+  showResponsiveControls={['fullScreen']}
+  screenContainer
+  height="650px"
+  width="100%"
+>
+  <InlineNotificationExample />
+</Example>
+
+## When to use
+
+Consider using an inline notification when presenting:
+
+- **Feedback** &ndash; Providing feedback to users about their actions. For
+  example, if a user submits a form requiring server-side validation, an
+  inline notification should be used to display and summarize form-level errors.
+- **Significant status updates** &ndash; Delivering significant status updates
+  about a task. For example, if a user initiated an ansynchronous process such
+  as a device update, a notification might provide the status update upon.
+  "Success - all devices have updated to the latest version."
+- **Promotions** &ndash; Calling a user's attention to such as new features, tips,
+  best practices, or other announcements.
+
+## When to consider something else
+
+Inline notifications are quite flexible, however consider using the alternatives
+for the following scenarios:
+
+- **Low severity information** &ndash; If communicating low severity information
+  which does not need to persist or require the user's attention, consider using
+  a [toast notification](/templates/toast-notifications).
+- **High attention information** &ndash; Consider using [global banner notification](/templates/global-banner-notifications)
+  when communicating information which might result in loss of functionality or
+  requires the user's attention and/or action. Examples might include a system
+  outage, critical error, or expiring subscription or license. [Global banner notifications](/templates/global-banner-notifications)
+  are persistent and require user action to acknowledge or dismiss.
+- **Immediate feedback** &ndash; When providing immediate feedback to a user's
+  action, a variety of tools are likely more appropriate.
+
+  - When a user is awaiting a response after clicking an action button, use the
+    [busy button]() which provides in-progress and success animations. If the
```

Comment:
FYI I created an issue to add a button busy example to the site https://github.com/grommet/hpe-design-system/issues/3265